Source/Core/Core/HW/EXI/EXI.cpp

```diff
@@ -62,6 +62,21 @@ void AddMemoryCards(int i)
 
   g_Channels[i]->AddDevice(memorycard_device, 0);
 }
+
+void AddSP1Device()
+{
+  TEXIDevices sp1_device = EXIDEVICE_BASEBOARD;
+  if (SConfig::GetInstance().bTri)
+  {
+    sp1_device = EXIDEVICE_BASEBOARD;
```

Comment:
this if-else can be collapsed down to the else case, since it's already assigned.